main.js

```diff
@@ -0,0 +1,51 @@
+// Create and show categories
+
+let categoriesConteiner = document.getElementById("categories");
+let searchProduct = document.getElementById("product-list");
```

Comment:
I'd rename the element:
searchProduct -> foundProductListEl

Reply:
I'd too.Done